extensions/cornerstone/src/Viewport/OHIFCornerstoneViewport.tsx

```diff
@@ -339,7 +339,7 @@ const OHIFCornerstoneViewport = React.memo(props => {
         elementEnabledHandler
       );
     };
-  }, []);
+  }, [viewportIndex]);
```

Comment:
The `viewportIndex` for a `viewportId` can change and when it does the viewport needs to be disabled.

Reply:
@wayfarer3130

Reply:
Why does it need to be disabled?  What if the viewport is just moving positions on screen - for example: